03/app.js

```diff
@@ -0,0 +1,9 @@
+const btnContainer = document.querySelector(".btn-actions");
+
+btnContainer.addEventListener("click", (e) => {
```

Comment:
W zadaniu było napisane, że "tylko raz" więc należałoby odpiąć event dla klikniętego już elementu. 
W tym rozwiązaniu będzie to ciężkie do zrealizowania (jeśli masz propagację, która przechwytuje wszystkie zdarzenia).